src/pudl_rmi/deprish.py

```diff
@@ -874,7 +874,7 @@ def _check_common_allocation(self, df_w_tots, split_col):
 
 def _calculate_rate_cols(
     df_to_fill: pd.DataFrame,
-    suffix: Literal['',  f'_w{COMMON_SUFFIX}']
+    suffix: Literal['', f'_w{COMMON_SUFFIX}']
```

Comment:
Should we add an assert in the function given we aren't running mypy?

Reply:
In theory maybe? But I think there's a ton of these annotations floating around like this unchecked right now. They help for doing tab completion, generating fuller annotations in docstrings etc. Really at some point we should think about how to turn mypy on.